app/models/incomeSourceDetails/TaxYear.scala

```diff
@@ -16,20 +16,29 @@
 
 package models.incomeSourceDetails
 
-import java.lang.reflect.UndeclaredThrowableException
 import scala.util.Try
 
-case class TaxYear(startYear: Int, endYear: Int)
+case class TaxYear(startYear: String, endYear: String)
```

Comment:
if we can keep this, as this more accurate description of this type
`case class TaxYear(startYear: Int, endYear: Int)`